docs/packages/pkg/lehrfempp.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    LehrFEMpp
+
+.. index::
+  single: math ; lehrfempp
+
+.. _pkg.lehrfempp:
+
+lehrfempp
+=========
+
+-  `Official <https://github.com/craffael/lehrfempp>`__
+-  `Hunterized <https://github.com/hunter-packages/lehrfempp>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/lehrfempp/CMakeLists.txt>`__
+-  Added by `craffael <https://github.com/craffael>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```

Comment:
`s,N,1629,` see:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#pull-requests
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#add-documentation

Reply:
fixed, I really forgot about it in the end even though I read about this step at least 3 times....